### PR TITLE
MAINT/ENH: Use scikit-bio>=0.4.0 and scipy>=0.17.0

### PR DESCRIPTION
**Pull request by ElDeveloper**
Change the usage of OrdinationResults to match its new API.

Use scipy's procrustes function instead of scikit-bio's. This is the first of many steps towards PY3 compatibility.

### CODE REVIEW

**Comment by ElDeveloper** [comment]
Tests are passing, this should be ready for review.

**Comment by ElDeveloper** [comment]
@ebolyen or @jairideout if you guys have a chance can you have a quick look?

File: .travis.yml [hunk 1]
```diff
@@ -14,10 +14,10 @@ before_install:
   # Update conda itself
   - conda update --yes conda
 install:
-  - conda create --yes -n env_name python=$TRAVIS_PYTHON_VERSION pip numpy${NUMPY_VERSION} scipy matplotlib openpyxl=1.8.2 pandas nose flake8 pep8 ipython-notebook
+  - conda create --yes -n env_name python=$TRAVIS_PYTHON_VERSION pip numpy${NUMPY_VERSION} 'scipy>=0.17.0' matplotlib openpyxl=1.8.2 pandas nose flake8 pep8 ipython-notebook
```
Comment (jairideout) [inline .travis.yml]: Minor: you can obtain scikit-bio from Anaconda.org by passing `-c biocore`

**Comment by jairideout** [comment]
@ElDeveloper this looks good, was there anything in particular you'd like me to review?

**Comment by ElDeveloper** [comment]
@jairideout, thanks for the review, mainly was looking for comments regarding our usage of the OrdinationResults API, and the IO bits that concern to these changes. Thanks for having a look!

**Comment by mortonjt** [comment]
Looking good! 

Now is this close to merging?  Because I'm not able to get a basic test run working ...

**Comment by jairideout** [comment]
@ElDeveloper looks fine to me, :shipit:

**Comment by ElDeveloper** [comment]
@jairideout, thanks for the review!!

> Now is this close to merging? Because I'm not able to get a basic test run working ...

This is ready to be merged. Can you tell me what's failing in your system?

**Comment by mortonjt** [comment]
Having trouble importing for some reason ...

```python
ImportError                               Traceback (most recent call last)
<ipython-input-1-f25046316ad1> in <module>()
      5 from skbio.stats.composition import closure, ilr, multiplicative_replacement, clr, centralize
      6 from biom import load_table
----> 7 from emperor import Emperor
      8 from skbio.stats.ordination import OrdinationResults
      9 

/Users/mortonjt/Documents/UCSD/research/emperor/emperor/__init__.py in <module>()
      8 __version__ = "0.9.51-dev"  # noqa
      9 
---> 10 from emperor.core import Emperor
     11 
     12 __all__ = ['Emperor', 'biplots', 'format', 'filter', 'parse', 'sort', 'util']

/Users/mortonjt/Documents/UCSD/research/emperor/emperor/core.py in <module>()
     29 from jinja2 import Template
     30 
---> 31 from emperor.util import get_emperor_support_files_dir
     32 
     33 # we are going to use this remote location to load external resources

/Users/mortonjt/Documents/UCSD/research/emperor/emperor/util.py in <module>()
      9 
     10 from numpy import ndarray, ones, zeros, vstack
---> 11 from string import strip
     12 
     13 from os import makedirs, listdir

ImportError: cannot import name 'strip'
```

**Comment by ElDeveloper** [comment]
I see, the problem is that you are using Python 3, this PR is not
intended to bring the code-base up to PY3K, instead it is intended to
add support for a newer version of scipy/scikit-bio.

**Comment by mortonjt** [comment]
OK, I misunderstood this PR.

**Comment by mortonjt** [comment]
Looks good to me.

**Comment by mortonjt** [comment]
Shall we shoot for PY3 compatibility next?  There is very little python in the current code base, so it shouldn't be that big.

**Comment by ElDeveloper** [comment]
Thanks for merging!

I think that's a *great* idea. Here's what I think makes the most sense:
(1) remove unnecessary code from format.py (most of it is now being done
through templating using jinja2), (2) remove all code from
make_emperor.py, (3) move all the code to PY3K, (4) first version of the
emperor CLI using click. For reviewing simplicity, I think we should
follow that order in separate PRs. What do you guys think?

**Pull request merged**